reference/resources/felixconfig.md

```diff
@@ -123,6 +124,25 @@ spec:
 | min      | Minimum index to use | 1-250             | int    |
 | max      | Maximum index to use | 1-250             | int    |
 
+#### AWS IAM Role/Policy for source-destination-check configuration
+
+Setting `awsSrcDstCheck` to `Disable` will automatically disable source-destination-check on EC2 instances in a cluster, provided necessary IAM roles and policies are set. One of the policies assigned to IAM role of cluster nodes must contain statement similar to the following:
+
+```
+{
+    "Effect": "Allow",
+        "Action": [
+            "ec2:DescribeInstances",
+            "ec2:ModifyNetworkInterfaceAttribute"
+        ],
+    "Resource": "*"
+}
+```
+
+If there are no policies attached to node roles containing the above statement, attach a new policy. For example, if a node role is `test-cluster-nodeinstance-role`, click on the IAM role in AWS console. This will present `Permission policies` list. Add a new inline policy by adding the above block of statement to statement of new policy JSON definition. For detailed information, refer to {% include open-new-window.html text='AWS documentation' url='https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html?icmpid=docs_iam_console' %}.
```

Comment:
...In the `Permission policies` list, add a new inline policy with the above statement to the new policy JSON definition. For detailed information, see  {% include open-new-window.html text='AWS documentation' url='https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html?icmpid=docs_iam_console' %}.